iip-12.md

```diff
@@ -0,0 +1,323 @@
+```
+IIP: 12
+Title: Support staking transactions via JSON-RPC
+Author: Haixiang Liu(haixiang@iotex.io), Dustin Xie (dustin.xie@iotex.io)
+Status: Final
+Type: Standards Track
+Category: Core
+Created: 2022-05-25
+```
+
+## Abstract
+
+This proposal explains the mechanism of supporting staking transactions via JSON-RPC 
```

Comment:
technically, web3.js protocol uses JSON-PRC as its underlying data format/protocol, but web3.js protocol is more known for it's the protocol that Ethereum used, which is the reason we want to implement it (but not other JRPC proctol) on our chain.

so I would say via `web3.js protocol (Ethereum-compatible)`